docker/Dockerfile

```diff
@@ -0,0 +1,65 @@
+ARG ISAAC_SIM_VERSION=2023.1.1
+
+# Download base image
+FROM nvcr.io/nvidia/isaac-sim:${ISAAC_SIM_VERSION}
+
+# Arguments for building
+ARG USERID
+ARG USER
+
+# Setup environment
+ENV TERM linux
+ENV DEBIAN_FRONTEND noninteractive
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+
+# Add ROS 2 key
+RUN curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key -o /usr/share/keyrings/ros-archive-keyring.gpg
+RUN echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu $(. /etc/os-release && echo $UBUNTU_CODENAME) main" | tee /etc/apt/sources.list.d/ros2.list > /dev/null
+RUN apt update
+
+# Copy installation scripts, and perform the dependency installation.
+COPY docker/install.sh .
+COPY docker/requirements.txt .
+RUN ./install.sh
+RUN rm install.sh requirements.txt
+
+# Adds the rosdep update to the entrypoint.
+RUN rosdep init && rosdep update
+
+# Create a user with passwordless sudo
+RUN groupmod -g 1001 ubuntu \
+  && usermod -u 1001 -g 1001 ubuntu
+RUN adduser --uid $USERID --gecos "ekumen developer" --disabled-password $USER
```

Comment:
Failed with this:
```
 => ERROR [12/25] RUN adduser --uid 1001 --gecos "ekumen developer" --disabled-password franco                                                                                                                                         0.6s 
------                                                                                                                                                                                                                                      
 > [12/25] RUN adduser --uid 1001 --gecos "ekumen developer" --disabled-password franco:                                                                                                                                                    
0.580 adduser: The UID 1001 is already in use.                                                                                                                                                                                              
------                                                                                                                                                                                                                                      
Dockerfile:32
--------------------
  30 |     RUN groupmod -g 1001 ubuntu \
  31 |       && usermod -u 1001 -g 1001 ubuntu
  32 | >>> RUN adduser --uid $USERID --gecos "ekumen developer" --disabled-password $USER
  33 |     RUN adduser $USER sudo
  34 |     RUN echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
--------------------
```

Reply:
I ended up removing the UID, seems like it's not needed. I built and ran everything without it. 

Mind giving it a shot now?